pkg/hoist/hoist_launchable.go

```diff
@@ -223,19 +215,10 @@ func (hl *Launchable) Executables(
 	var executables []launch.Executable
 	for _, service := range services {
 		serviceName := fmt.Sprintf("%s__%s", hl.Id, service.Name())
-		execCmd := []string{
-			hl.P2exec,
-			"-n",
-			"-u",
-			hl.RunAs,
-			"-e",
-			hl.ConfigDir,
-			"-l",
-			hl.CgroupConfigName,
-			"-c",
-			hl.Id,
-			filepath.Join(serviceDir, service.Name()),
-		}
+		execCmd := append(
+			[]string{hl.P2exec},
+			append(hl.p2ExecArgs(), filepath.Join(serviceDir, service.Name()))...,
```

Comment:
lmk if i should break this up into multiple lines or if i'm missing some special Go way of doing this